setup.py

```diff
@@ -37,7 +37,7 @@ def get_version(rel_path):
 
 vis_requirements = requirements + [
     "matplotlib>=3.5.1",
-    "plotly>=5.3.1",
+    "plotly>=5.5.0",

```

Comment:
Todo - will need to clean :P you can give it a try unless you're stuck

Reply:
Is there an issue with this?